### PR TITLE
Update code file

### PR DESCRIPTION
**Pull request by ghost**
Update Variable and funtion notation

### CODE REVIEW

**Review by omar-saadoun — CHANGES_REQUESTED** [review]
Please fix this variable too : 

`stranswer`

as well as syntax following the same indications in this entrypoint:

 ```
payable stateful entrypoint **quest_answer**(**quest** : string, **answ** : string) : bool = 
    let **val** = Call.value
    if(val > 0)
      false
    else
      put(state{question_answer[**quest**] = **answ** })
      true
```

**Comment by mgomez-code** [comment]
update syntax
payable stateful entrypoint **quest_answer**(**quest** : string, **answ** : string) : bool = 
   let **val** = Call.value
   if(val > 0)
     false
   else
     put(state{question_answer[**quest**] = **answ** })
     true

File: oracle_en/index.js [hunk 1]
```diff
@@ -1,87 +1,99 @@
...
-    if(val > 0)
-      false
-    else
-      put(state{question_answer[quest] = answ })
-      true
-  entrypoint getQuestion(  								
+
+  stateful entrypoint quest_answer(quest : string, answ : string) : bool = 
+    put(state{question_answer[quest] = answ })
+    true
```
Comment (thepiwo) [inline oracle_en/index.js:32]: why is there the need to return true?

Comment (mgomez-code) [reply oracle_en/index.js:32]: The bolean is used for validation purposes

Comment (thepiwo) [reply oracle_en/index.js:32]: why is that validation needed, either it return or it aborts, the boolean is not needed then

**Pull request closed without merging**